app/docs/md/learn/components/image.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Image
+---
+
+Images can enrich web content, but they can also introduce performance issues, from reducing a page’s time to [Largest Contentful Paint](https://web.dev/lcp/) to introducing [Cumulative Layout Shift](https://web.dev/cls/).
+
+The Enhance image component is designed to help alleviate these issues for images stored in your project’s [public folder](/docs/learn/starter-project/public), by providing: 
+
+<dl class='pl2 pl4-lg'>
+  <dt class='font-medium'>Configurable image optimization</dt>
+  <dd>Generate alternative formats, set quality levels for lossy formats, and generate different image sizes on the fly.</dd>
+  
+  <dt class='font-medium mt0'>Responsive images</dt>
+  <dd>The <a href='https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture'>picture element</a> offers a lot of power in delivering images tailored to different media conditions, but it can be challenging to implement correctly. The image component abstracts and simplifies this API to make responsive images a breeze.</dd>
+
+  <dt class='font-medium mt0'>Generated sizing attributes</dt>
+  <dd>For each image variant you provide, the image component will generate corresponding <code>width</code> and <code>height</code> attributes to prevent layout shift from occurring.</dd>
+
+  <dt class='font-medium mt0'>Lazy loading</dt>
+  <dd>Images that aren’t critical for a page’s initial render can be lazy loaded to improve the <a href='https://developer.mozilla.org/en-US/docs/Web/Performance/Critical_rendering_path'>critical rendering path</a>.</dd>
+</dl>
+
+## Usage
+
+### enhance-image
+This parent element is used to specify your image’s source, its loading strategy, and any <code>alt</code> text you wish to provide.
+
+#### Attributes
+
+<dl class='pl2 pl4-lg'>
+  <dt class='font-bold'><code>src</code> <span class='font-normal'>required</span></dt>
+  <dd>The path to your source image. In order to take advantage of image optimization, your image <em>must</em> reside in your project’s public folder. This source will also be used as the fallback image in case none of the provided <code>enhance-image-variant</code>s’ media conditions are matched by the client.</dd>
```

Comment:
Is the ' on the wrong side of the `s` for `enhance-image-variant`

Reply:
Haha, good question, I had to think about it for a second. It's a plural possessive — makes more sense without the formatting:

> …in case none of the provided enhance-image-variants’ media conditions are matched…